docs/glueops-platform-administrator/configuration/export-logs.md

```diff
@@ -0,0 +1,292 @@
+---
+id: export-logs
+title: Exporting Logs from GlueOps
+---
+
+# Export/Ship your logs to an external sink
+
+:::danger
+These docs are beta.
+:::
+ 
+We use the [fluent-operator](https://github.com/fluent/fluent-operator) to ship your logs however the version we run is not always the latest so it's best to verify the correct CRDs to use.
```

Comment:
should we link to the version we're using so they don't have to look for it?

Reply:
We will have to remember to update the docs each time we update it. I figure keeping it vague for now felt best

Reply:
I wasn't clear - I meant just linking them to platform helm chart so they can check: https://github.com/GlueOps/platform-helm-chart-platform/blob/131b5cd87bb6b062c51b1f6fbe3f24ea84b2f28e/templates/application-fluent-operator.yaml#L29

I'm good with vague as well, but I don't think they have a good way to check without referencing the repo